GameData/Skopos/Contracts/Level1Comsats/l1_us_tracking_ships.cfg

```diff
@@ -12,8 +12,8 @@ CONTRACT_TYPE {
   sortKey = 280
   title = US Tracking Ships - Level 1
   // Mostly pilfered from NASA SP-32 p. 740.
-  description = <b>Program: First Generation Telecommunications Satellites (Skopos Beta)<br>Type: <color=blue>Optional</color></b><br><br>We would like to create "Ground stations at sea", ships equipped with unified S-band antennas to relay data from manned and unmanned scientific spacecraft. However, a ship at sea cannot have a landline connection to mission control, and existing HF radio systems are unreliable and low quality. Instead, these ships have had a satellite communications terminal installed, allowing them to relay data back to mission control via communications satellites.
-  notes = The tracking ships USNS Vanguard, Redstone and Mercury have been positioned in the Atlantic and Pacific near the equator. If they are provided with a link to shore, they will be able to use their unified S-band systems to function as a ground station. Whether you achieve this with a MEO or LEO constellation or a single GEO satellite is up to you.
```

Comment:
Same here, this should probably say "tracking station" to indicate that it will in fact function as one, not purely as a communications station.